test/ui-test/testSuites/suite_status/tst_wallet/test.feature

```diff
@@ -58,6 +58,24 @@ Feature: Status Desktop Wallet
         When the user adds a saved address name <name> and address <address>
         Then the name <name> is in the list of saved addresses
 
+    	Examples:
+      	  | name | address 								      |
+      	  | one  | 0x8397bc3c5a60a1883174f722403d63a8833312b7 |
+
+    Scenario Outline: User can edit a saved address
+        When the user adds a saved address name <name> and address <address>
+        And the user edits a saved address with name <name> to <new_name>
+        Then the name <new_name><name> is in the list of saved addresses
```

Comment:
Is this a mistake (double name)?

Reply:
This is because the input preprend by default.